arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -961,7 +937,6 @@ bool SynchronizeShard::first() {
           catchupWithReadLock(ep, *collection, clientId, leader, lastTick, tailingSyncer);
 
       if (!tickResult.ok()) {
-        tailingSyncer->unregisterFromLeader();
```

Comment:
Is it just me or is this call to `unregisterFromLeader` now missing...?

Reply:
It is missing now, but I added a comment somewhere in this PR stating that the method will be called in the syncer's destructor anyway.

Reply:
👍